src/test/java/wooteco/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -0,0 +1,160 @@
+package wooteco.subway.acceptance;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import wooteco.subway.dto.LineResponse;
+
+@DisplayName("노선 관련 기능")
+public class LineAcceptanceTest extends AcceptanceTest {
+
+    @DisplayName("노선을 생성한다.")
+    @Test
+    void createLine() {
+        final ExtractableResponse<Response> response = createLine("신분당선", "bg-red-600");
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        assertThat(response.header("Location")).isNotBlank();
+    }
+
+    @DisplayName("기존에 존재하는 노선 이름으로 노선 생성 시도 시 Bad request가 응답된다.")
+    @Test
+    void createLineWithDuplicateName() {
+        createLine("강남역", "bg-red-600");
+        final ExtractableResponse<Response> response = createLine("강남역", "bg-red-600");
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
+    }
+
+    @DisplayName("노선을 조회한다.")
+    @TestFactory
+    Stream<DynamicTest> getLinesTest() {
+        final ExtractableResponse<Response> createResponse1 = createLine("1호선", "bg-red-600");
+        final ExtractableResponse<Response> createResponse2 = createLine("2호선", "bg-green-600");
+
+        return Stream.of(
+                DynamicTest.dynamicTest("생성된 노선 목록을 불러온다", () -> {
+                    final ExtractableResponse<Response> response = getLines();
+
+                    assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+                }),
+                DynamicTest.dynamicTest("생성된 노선이 저장한 노선과 일치한지 확인한다.", () -> {
+                    final ExtractableResponse<Response> response = getLines();
+                    final List<Long> expectedLineIds = Stream.of(createResponse1, createResponse2)
+                            .map(it -> Long.parseLong(it.header("Location").split("/")[2]))
+                            .collect(Collectors.toList());
+                    final List<Long> resultLineIds = response.jsonPath().getList(".", LineResponse.class).stream()
+                            .map(it -> it.getId())
+                            .collect(Collectors.toList());
+
+                    assertThat(resultLineIds).containsAll(expectedLineIds);
+                })
```

Comment:
마이너하지만 

```suggestion
                DynamicTest.dynamicTest("생성된 노선 목록을 불러온다", () -> {
                    final ExtractableResponse<Response> response = getLines();

                    assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
                }),
                
                DynamicTest.dynamicTest("생성된 노선이 저장한 노선과 일치한지 확인한다.", () -> {
                    final ExtractableResponse<Response> response = getLines();
                    final List<Long> expectedLineIds = Stream.of(createResponse1, createResponse2)
                            .map(it -> Long.parseLong(it.header("Location").split("/")[2]))
                            .collect(Collectors.toList());
                    final List<Long> resultLineIds = response.jsonPath().getList(".", LineResponse.class).stream()
                            .map(it -> it.getId())
                            .collect(Collectors.toList());

                    assertThat(resultLineIds).containsAll(expectedLineIds);
                })
```

중간에 개행을 넣어주면 가독성이 좋아질 것 같네요